RHDEVDOCS-6891: CQA 2.0: Work with shared resources (#102701)
openshift/openshift-docs: modules/ephemeral-storage-creating-a-shared-secret-instance.adoc

Reviewer: Just checking because this seems to be a CQA PR and you are fixing callouts: I suppose the block titles will be removed in another PR?

Author: Yes

= Creating a SharedSecret custom resource

[role="_abstract"]

> Create a `SharedSecret` custom resource (CR) to share a `Secret` object across namespaces. By creating a SharedSecret custom resource (CR), you can share a Secret object across namespaces, ensuring that sensitive configuration data remains consistently accessible cluster-wide.

Reviewer: ```suggestion
Create a `SharedSecret` custom resource (CR) to share a `Secret` object across namespaces. By creating a `SharedSecret` CR, you can share a `Secret` object across namespaces, ensuring that sensitive configuration data remains consistently accessible cluster-wide.
```

apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: share-etc-pki-entitlement
  namespace: openshift-config-managed
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: share-etc-pki-entitlement 
subjects:
  - kind: ServiceAccount
    name: csi-driver-shared-resource
    namespace: openshift-builds
----
`metadata.name`:: Specifies the name of the `RoleBinding` CR.
`roleRef.name`:: Specifies the name of the `Role` CR.

> `subjects.name`:: Specifies the name of the `ServiceAccount` CR.

Reviewer: This again should be in the bulleted list (the definition list is used for only user-replaced values in the CCS style guide:

```suggestion
* `metadata.name` defines the name of the `RoleBinding` CR.
* `roleRef.name` defines the name of the `Role` CR.
* `subjects.name` defines the name of the `ServiceAccount` CR.
```

    verbs:
      - use
----
`name`:: Name of the `ClusterRole` CR.

.Verification

. Run the following command to verify that the required `Role` is created:
+
[source,terminal]
----
$ oc get role share-etc-pki-entitlement -n openshift-config-managed
----
+

> .Example output

Reviewer: To not add block titles
```suggestion
Example output:
+
```
